### PR TITLE
Fix long_description_content_type in setup.cfg

### PR DESCRIPTION
**Pull request by avalentino**
Fix the incorrect values of `long_description_content_type` in `setup.cfg`.
It is now set to `text/x-rst`.
Unfortunately `twine check` still complains for the `missing long_description_content_type`

### CODE REVIEW

**Review by robbmcleod — COMMENTED** [review]
I'm confused. The README is markdown, not RST. I think PyPi had issues because I uploaded the wheels first, then when I tried to upload the sdist it complained it couldn't find the `long_description` file. So I edited `site.cfg` but then it perhaps had some issue with the existing wheels having the incorrect filename.

**Comment by avalentino** [comment]
OK, now I'm confused as well.
In master I see only one README file, it has `.rst` extension, and indeed it seems to be a RST file.
That one is the file that I have pointer to in the `setap.cfg` in the `long_description` field.
Do I miss something?

File: setup.cfg [hunk 1]
```diff
@@ -10,7 +10,7 @@ maintainer = Robert A. McLeod
 maintainer_email = robbmcleod@gmail.com
 url = https://github.com/pydata/numexpr
 long_description = file: README.md
```
Comment (stephan-cr) [inline setup.cfg:12]: This line should be `long_description = file: README.rst`.

Comment (avalentino) [reply setup.cfg]: Thanks @stephan-cr

**Comment by stephan-cr** [comment]
@robbmcleod: @avalentino is right, the README.rst is ReStructured Text, the file RELEASING.md is Markdown.

**Pull request merged**